src/foam/core/stdlib.js

```diff
@@ -863,6 +863,13 @@ foam.LIB({
         }
         return ( type[uid] || opt_defaultMethod ).apply(this, arguments);
       };
+      var toString = f.toString.bind(f);
```

Comment:
Add some comments to explain why this is being done.

Reply:
Done. Does the comment make sense?

Also, what do you think of the fallback for foam.json.stringify not existing? I'm torn on whether to send the original toString's result or just returning something like 'function() {}'. What do you think?